app/views/krikri/harvest_sources/index.html.erb

```diff
@@ -21,5 +26,6 @@
 
 <%= link_to 'New',
     new_institution_harvest_source_path(institution_id: @institution.id)
-%>
+%> | 
```

Comment:
I think we had decided to wait on all non-urgent styling. We can fix it now if you would like, but I would be okay with leaving it until we start styling work in earnest.